handbook/engineering/https_dev_instance.md

```diff
@@ -0,0 +1,173 @@
+# How to run an HTTPS reverse proxy for a Sourcegraph instance on a local dev machine
+
+> NOTE: These instructions have been verified on a Mac running Catalina (the most common dev machine configuration).
+> It should fundamentally work the same on Linux with commands tailored accordingly. All the big pieces work
+> on both Linux and Mac.
+
+## Method One
```

Comment:
`ngrok` has really bad latency for me, so I'd suggest that "Method Two" should be the recommended approach by swapping the two

Reply:
okay, will do